config.js

```diff
@@ -25,6 +25,12 @@ module.exports = {
     dev: {
         proxy: 'socks://localhost:12345',
     },
+    openId: {
```

Comment:
Configuration file should not be pushed to the inventory but only a `default-config.js` similar to the other AliceO2 projects. 
This is to preevent pushing information such as this one to GitHub as we are open source.

Please remove the configuration of openid from the repository or modify it to a mocked one.